bucket/aws-nosql-workbench.json

```diff
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://aws.amazon.com/dynamodb/nosql-workbench/",
+    "description": "NoSQL database designer for Amazon DynamoDB and Amazon Keyspaces",
+    "version": "2.2.0",
+    "license": "Proprietary",
+    "architecture": {
+        "64bit": {
+            "url": "https://s3.amazonaws.com/nosql-workbench/NoSQL%20Workbench-win-2.2.0.exe#/dl.7z",
+            "hash": "sha512:4003a51ac32e64c9afd06ea6699ace5ff71af40f65ca0d047414bf8891992c13e5b125d29ab48cc209b7aaddedc52bbead0c49e5270129c0de2929e8d1c93c98",
+            "pre_install": [
+                "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" -DestinationPath \"$dir\"",
+                "Remove-Item -Path \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "bin": "NoSQL Workbench.exe",
```

Comment:
What are available parameters?

Reply:
I could not find any available command-line parameters. It's an Electron app, so it may possibly have some of these: https://www.electronjs.org/docs/api/command-line-switches

However, AWS has not released the source code for this app, so impossible to see for certain.

Reply:
Should I remove the shim from the manifest?